Week-6/6.6/assignment6.6.js

```diff
@@ -0,0 +1,28 @@
+function threesum(arr, sum) {
+    let arr1 = arr.slice();
```

Comment:
@mr-rego can you think about reducing the time complexity here, because the time complexity here is O(n^3). Is there any alternative ?

Better way to solve the problem - https://www.geeksforgeeks.org/find-a-triplet-that-sum-to-a-given-value/